### ISSUE TITLE
White page after failed login

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#4395]
I just tried to trigger a notification bubble by logging in with wrong credentials. While doing so I discovered that a failed login results in a blank white page. Expected behavior: reload/return to the login page and show a notification about the failed login.

Btw. notifications seem to be broken too.

**Comment by annando** [comment]
Have you updated the addons as well? And are you on the "develop" level there as well? Have you run the composer?

**Comment by AlfredSK** [comment]
Yes. Yes and yes.

**Comment by annando** [comment]
Do you see something in the PHP error log? (And does this log file has the permissions for both the frontend and the backend?)

**Comment by AlfredSK** [comment]
I need to check that. But what do you mean by "permissions for both the frontend and the backend"? PHP is running as Apache mod_php. So it always is running as user "www-data". That user can write to the logfile. Is there anything else?

**Comment by zeroadam** [comment]
I just tested this on my node, by accident, as a result of typing too fast. I was presented with the notification bubble and a visible login screen as expected. Frio as the main theme.

**Comment by AlfredSK** [comment]
Hmm. Doesn't look like an error to me. Or is it?

````[04-Feb-2018 13:21:08 UTC] PHP Fatal error:  Uncaught Exception: Not enough information to authenticate in /var/www/html/libranet/src/Model/User.php:145          Stack trace:
#0 /var/www/html/libranet/src/Module/Login.php(106): Friendica\Model\User::authenticate('herbert12', 'herbert12')
#1 /var/www/html/libranet/index.php(343): Friendica\Module\Login::post()
#2 {main}                                               thrown in /var/www/html/libranet/src/Model/User.php on line 145````

**Comment by zeroadam** [comment]
That line is a throw error line giving the message of 'Not enough information to authenticate'

**Comment by MrPetovan** [comment]
This happens when we can't retrieve the necessary information from the `user` table. This seems related to #4396, both are simple queries to the `user` table that are failing unexpectedly.

**Comment by MrPetovan** [comment]
Duplicated by #4426

**Comment by MrPetovan** [comment]
Fixed by #4429
